### DISCUSSION TITLE
Disabling TLS possible?

### DISCUSSION THREAD

**Discussion by fietsknecht** [haraka/Haraka#3250] [Q&A]
I want to use Haraka as a mail server for vintage computers that cannot do secure connections. Tried to do auth via auth_flat_file but it requires tls. Any workaround?

**Comment by msimerson** [comment]
Hack your local copy of Haraka and remove the requirement. There is a 0% chance we'll support this feature.

**Comment by fietsknecht** [comment]
Sounds fair. Just asking, as Postfix allows diabling tls easily.
